psi4/src/psi4/libfunctional/LibXCfunctional.cc

```diff
@@ -215,14 +216,22 @@ double LibXCFunctional::query_density_cutoff() { return xc_functional_->dens_thr
 void LibXCFunctional::set_omega(double omega) {
     omega_ = omega;
     user_omega_ = true;
-    if ((xc_func_name_ == "XC_GGA_X_WPBEH") || (xc_func_name_ == "XC_GGA_X_HJS_PBE") ||
-        (xc_func_name_ == "XC_HYB_GGA_XC_LRC_WPBEH") || (xc_func_name_ == "XC_HYB_GGA_XC_WB97X") ||
-        (xc_func_name_ == "XC_HYB_GGA_XC_WB97") || (xc_func_name_ == "XC_HYB_GGA_XC_WB97X_V") ||
-        (xc_func_name_ == "XC_HYB_GGA_XC_WB97X_D") || (xc_func_name_ == "XC_HYB_MGGA_X_M11")) {
+
+    // See if Libxc has an omega parameter for the functional
+    int npars = xc_func_info_get_n_ext_params(xc_functional_.get()->info);
+    bool match = false;
+    for (int ipar = 0; ipar < npars; ipar++) {
+        const char* name = xc_func_info_get_ext_params_name(xc_functional_.get()->info, ipar);
+        if (stricmp(name, "_omega") == 0) match = true;
```

Comment:
/home/vsts/work/1/s/psi4/src/psi4/libfunctional/LibXCfunctional.cc:225:13: error: ‘stricmp’ was not declared in this scope
         if (stricmp(name, "_omega") == 0) match = true;
             ^~~~~~~
```
/home/vsts/work/1/s/psi4/src/psi4/libfunctional/LibXCfunctional.cc:225:13: note: suggested alternative: ‘strncmp’
         if (stricmp(name, "_omega") == 0) match = true;
             ^~~~~~~
             strncmp
```

Azure error